zhaquirks/const.py

```diff
@@ -110,6 +110,7 @@
 UNKNOWN = "Unknown"
 VALUE = "value"
 ZHA_SEND_EVENT = "zha_send_event"
+ZONE_STATUS_CHANGE_COMMAND = 0
```

Comment:
Despite being the same, I would prefer the `0x00` value from the command declaration:
https://github.com/zigpy/zigpy/blob/be5b2914a12ac6a19af80c5a83dab40d7957aa01/zigpy/zcl/clusters/security.py#L103

Reply:
Used `0x0000` now (https://github.com/zigpy/zha-device-handlers/pull/2231/commits/d0861d8fa7e899d4f0f58fbc2b62e75ef9a750e7) to fit the style of `ZONE_TYPE`/`ZONE_STATUS`, as they're both 4 digits.